src/settings.c

```diff
@@ -516,6 +533,40 @@ void load_settings(char *cmdline_config_path)
                 );
 
         }
+
+        {
+                char *c = option_get_string(
+                        "global",
+                        "mouse_left_click", "-left_click", "close_current",
```

Comment:
I guess, the only thing, which is still against current consistency: Please put the default value in the `defaults` struct in the file `config.h`. There is a full `settings_t` allocated and you can specify the default value there.

Use
```
char *c = option_get_string(
                        "global",
                        "mouse_left_click", "-left_click", NULL,
```

And then do it similarly to `markup_mode`:

https://github.com/dunst-project/dunst/blob/6d0e20e40b90dce4967cc66f40476b0e41c25542/src/settings.c#L160-L167